code/modules/surgery/surgery.dm

```diff
@@ -115,30 +115,30 @@
 	SSblackbox.record_feedback("tally", "surgeries_completed", 1, type)
 	qdel(src)
 
-/datum/surgery/proc/get_propability_multiplier(mob/user)
+/proc/get_speed_modifier(mob/M, mob/N)
 	var/propability = 0.3
-	var/turf/T = get_turf(target)
+	var/turf/T = get_turf(N)
 	var/selfpenalty = 0
 	var/sleepbonus = 0
-	if(target == user)
-		if(HAS_TRAIT(user, TRAIT_SELF_AWARE) || locate(/obj/structure/mirror) in view(1, user))
+	if(N == M)
```

Comment:
Those names are really unclear

Reply:
T is already claimed by the turf, so i just took the closest letter to M

Reply:
Why not use target and user?

Reply:
i honestly have no clue how it worked with target/user when it only had one parameter for user, and many other procs just use letters

Reply:
Longer and more descriptive names don't cost anything extra